iam.tf

```diff
@@ -0,0 +1,40 @@
+data "aws_iam_policy_document" "log_assume" {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["logs.${length(var.region) > 0 ? var.region: data.aws_region.default.name}.amazonaws.com"]
+    }
+  }
+}
+
+data "aws_iam_policy_document" "log" {
+  statement {
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents",
+      "logs:DescribeLogGroups",
+      "logs:DescribeLogStreams",
+      "logs:CreateLogStream",
+      "logs:DeleteLogStream",
+    ]
+
+    resources = [
+      "${aws_cloudwatch_log_group.default.arn}",
+      "${join(",", aws_cloudwatch_log_stream.default.*.arn)}",
```

Comment:
Is `join(","` deliberate?

Reply:
yes, for prevent mistakes in `json`

Reply:
explain what mistakes it prevents. it's not clear.

Reply:
element in `resources` should be separated by comma, I'm not sure that comma will be without `join()`